lib/image.js

```diff
@@ -44,6 +44,12 @@ function processBasicImageOperations(img, instructions) {
     // for reproducible png files
     img = img.define("png:exclude-chunks=date");
 
+    if(instructions.autoOrient){ // uses info in exif data
+        img - img.autoOrient();
```

Comment:
Shouldn't it be an equal sign?

```suggestion
        img = img.autoOrient();
```

Reply:
I think I changed it the moment you saw it!